setup.cfg

```diff
@@ -39,10 +39,11 @@ tests =
     pytest-black-ng>=0.4.1
     future>=0.18.2
     invenio-access>=4.0.0,<5.0.0
+    flask-admin>=1.0.0,<2.0.0
```

Comment:
why do we need flask-admin here?

Reply:
Oups I meant to explain that one. There is this test: https://github.com/inveniosoftware/invenio-files-rest/blob/master/tests/test_admin.py#L35 (and doctest of `invenio_files_rest/__init__.py`) that check these views are registered: https://github.com/inveniosoftware/invenio-files-rest/blob/master/setup.cfg#L72 . But flask-admin-2,x has changed the interface to `Admin(...)` in such a way that the call breaks. The cap should be done on invenio-admin to be honest, but it's faster to do here than wait for merge and release. I am not too sure about the state of invenio-admin either. I guess it's maintained for Invenio framework but InvenioRDM is not using it anymore.

Reply:
i ping Zach on discord. normally if something is related to `invenio-admin`i ping him.